apps/backoffice-v2/src/pages/Entity/components/Notes/hooks/schemas/note-schema.ts

```diff
@@ -0,0 +1,21 @@
+import { z } from 'zod';
+
+const BasicNoteSchema = z.object({
```

Comment:
This is the naming we use in other places for schemas used for extend.
```suggestion
const BaseNoteSchema = z.object({
```